src/redisClient.ts

```diff
@@ -157,6 +159,25 @@ export class RedisClient implements CommonClient {
     return await this.redis().hincrby(key, field, increment)
   }
 
+  async hincrBatch(key: string, incrementTuples: [string, number][]): Promise<[string, number][]> {
```

Comment:
I added the batch support into our client-wrapper, because the `withPipeline()` exposes the underlying lib, and I don't think that consumers of our wrapper should be using that (if avoidable).

The consuming logic (`redis*KeyValueDb.incrementBatch`) has also its own business translating the keys - so this separation of concerns (here we pipe, there we translate ids) is also a plus,